### PR TITLE
Readme enhancement

### PR DESCRIPTION
**Pull request by jnogol**
- More info on 'Crates and parts' section. 
- Added links to FIDO specification for 'Protocols' section. 
- Added links to different repo folders on 'Components' section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,21 @@ The fido-fdo-rs implements all core components of the FIDO Device Onboard Specif
...
-- fdo-client-linuxapp:
+- `fdo-client-linuxapp`: Performs TO0, TO1 and TO2 client side protocols.
+- `fdo-data-formats`: [DI, TO0, TO1, TO2]: Implements the different low-level messaging formats used.
+- `fdo-http-wrapper`: Helpers for HTTP operations in both FDO server and client.
+- `fdo-integration-tests`: In a surprising turn of events, this crate contains the integration testing.
+- `fdo-libfdo-data`: C wrapper around `fdo-data-formats`, allowing code in other languages to parse Ownership Vouchers, and possibly other data formats in the future.
+- `fdo-manufacturing-client`: Client side implementation of Device Initalize and Device Initialize over
+Untrusted Networks (DIUN) protocols.
+- `fdo-manufacturing-server`: Server side implementation of Device Initalize and Device Initialize over
+Untrusted Networks (DIUN) protocols.
```
[the remark this answers is not in the document]
Comment (nullr0ute) [reply README.md]: WFM

[the remark this answers is not in the document]
Comment (jnogol) [reply README.md]: Done.

**Review by nullr0ute — APPROVED** [review]
I think this is good, we'll need to expand it further so there will be more updates.

**Comment by puiterwijk** [comment]
@Mergifyio refresh

**Pull request merged**